build.gradle

```diff
@@ -225,7 +235,7 @@ curseforge {
 		id = '306612'
 		changelog = 'A changelog can be found at https://github.com/FabricMC/fabric/commits/master'
 		releaseType = 'beta'
-		addGameVersion '1.15-Snapshot'
+		addGameVersion '1.15.2'
```

Comment:
Why? this appears wrong to me. Maybe you've messed up git

Reply:
something screwed up and I need to go look into why it messed up (probably cherrypick it to fix)

Reply:
Hmm due to the tag changes, should I count on doing this in 1.15 or 1.16.

I know modmuss had a blast backporting the object builders when I did them for 1.16 instead.

Reply:
You probably shouldn't change the base branch of pr to 1.16; should change it back to 1.15, and cherry pick the squashed commit onto 1.16 and finally reset the head of `feature/registry-tags` to the repicked commit, then change the pr's base branch to 1.16